static/main.css

```
@@ -313,12 +344,30 @@ main.normalize {
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
 
-@media only screen and (max-width: 735px) {
+/* latin */
+@font-face {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-display: swap;
+    src: url("/fonts/poppins-v19-regular-latin.woff2") format("woff2");
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+@media only screen and (max-width: 920px) {
     .hero {
         flex-flow: column nowrap;
         text-align: center;
     }
 
+    .hero h1 {
+        font-size: 30px;
```

Please use a relative unit: 30px ≙ 1.875rem = 30px / 16px*

*Web browsers calculate the REM (the root em size) based on the root element size. The default for modern web browsers is 16px, so the conversion is SP_SIZE/16 = rem.

```suggestion
        font-size: 1.875rem;
```